core/controllers/ingest.py

```diff
@@ -83,7 +86,7 @@ def ingest(body: dict, excel_file: FileStorage) -> tuple[dict, int]:
             # TODO https://dluhcdigital.atlassian.net/browse/SMD-653: replace hardcoded dependencies with dependency
```

Comment:
Hmm that's odd - SMD-653 is still an outstanding TODO, not sure why the pull request for SMD-636 is listed in the pull requests for SMD-653. Maybe just because Cyrus added a comment to the PR description for SMD-636 mentioning that SMD-653 was a further piece of work required? See quote from SMD-636 PR description:

> TODO:
> Inject PF dependencies dynamically instead of hardcoding them see https://dluhcdigital.atlassian.net/browse/SMD-653